pkg/github/github.go

```diff
@@ -128,6 +132,14 @@ type Client interface {
 	) (*github.IssueComment, *github.Response, error)
 }
 
+// NewIssueOptions is a struct of optional fields for new issues
+type NewIssueOptions struct {
+	Milestone string   // Name of milestone to set
+	State     string   // open, closed or all. Defaults to "open"
```

Comment:
I'm wondering why we have this field. What is the use case behind creating an issue with the state `closed` or `all`?

Reply:
:100:  I was just porting it without thinking about the application. I'll drop it.